token/core/cmd/pp/dlog/gen.go

```diff
@@ -28,10 +28,10 @@ type GeneratorArgs struct {
 	// GenerateCCPackage is whether to generate the chaincode package
 	GenerateCCPackage bool
 	// Issuers is the list of issuers to include in the public parameters.
-	// Each issuer should be specified in the form of <MSP-Dir>:<MSP-ID>
+	// Each issuer should be specified in the form of <MSP-Dir>
 	Issuers []string
 	// Auditors is the list of auditors to include in the public parameters.
-	// Each auditor should be specified in the form of <MSP-Dir>:<MSP-ID>
+	// Each auditor should be specified in the form of <MSP-Dir>
 	Auditors []string
```

Comment:
Related to the comment above, the variable `Auditors []string` does not really suggest what is contains. You chance to give it a proper name :)

Reply:
what's wrong with it. Isn't the description of the variable informative enough?

Reply:
okay, I changed the comment